include/sdf/InterfaceElements.hh

```diff
@@ -169,57 +170,57 @@ class SDFORMAT_VISIBLE NestedInclude
   /// not end with a file extension (it will not end with an extension if it
   /// refers to a model package).
   /// \deprecated Use NestedInclude::Uri() instead
-  public: std::string uri SDF_DEPRECATED(12);
+  public: GZ_DEPRECATED(12) std::string uri;
```

Comment:
Should these say 13?

Reply:
oh, these were actually deprecated in fortress; I suppose they can be removed here. Do you mind if I do that in a separate pull request?

Reply:
Oh yeah I didn't pay attention to the diff. SGTM